scripts/project-setup-scripts/create-custom-iam-roles-for-projects.sh

```diff
@@ -0,0 +1,15 @@
+#create custom roles in each environment's google project
+
+# todo: convert this to Terraform as part of https://broadworkbench.atlassian.net/browse/CA-1195
+
+# org: test.firecloud.org
+gcloud iam roles create google-project-viewer --organization=400176686919 --file=google-project-viewer.yaml
+gcloud iam roles create google-project-owner --organization=400176686919 --file=google-project-owner.yaml
+
+# org: quality.firecloud.org
+gcloud iam roles create google-project-viewer --organization=206744735509 --file=google-project-viewer.yaml
+gcloud iam roles create google-project-owner --organization=206744735509 --file=google-project-owner.yaml
+
+# org: firecloud.org
+gcloud iam roles create google-project-viewer --organization=386193000800 --file=google-project-viewer.yaml
+gcloud iam roles create google-project-owner --organization=386193000800 --file=google-project-owner.yaml
```

Comment:
Same feedback here, my preference would be that the script take an `organization` parameter